b-locker-web/src/components/organisation/singlelocker/orgSingleLocker.tsx

```diff
@@ -0,0 +1,303 @@
+                                <div className="box-right-side">
+                                    <p>4</p>
+                                </div>
+
+                            </div>
+                            <div className="box3">
+                                <div className="box-left-side">
+                                    <img className="locker-icon" src={lockerIcon} alt='' />
+                                    <p>Total</p>
+                                </div>
+                                <div className="box-right-side">
+                                    <p>14</p>
+                                </div>
+                            </div>
+                            <div className="box4">
+                                <a href="#">
+                                    <div className="box-left-side">
+                                        <img className="unlock-icon" src={unlockIcon} alt='' />
+
+                                    </div>
+                                    <div className="box-right-side">
+                                        <p>Unlock All</p>
+                                    </div>
+                                </a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div className="wrapper">
+                    <div className="locker-panel">
+                        <h2>Locker Details</h2>
+                        <div className="locker-details">
+                            <div className="locker-details-content">
+                                <p>Locker ID: 1.1</p>
+                                <p>Locker number: 1</p>
+                                <p>Locker Location: Stadslab</p>
+                                <p>Current Owner: 0867973@hr.nl</p>
+                                <p>Current Status: Locked</p>
+                            </div>
+
+                        </div>
+                        <h2>Unlock Locker</h2>
+                        <div className="locker-unlock">
+                            <a href="#"><img className="logout-icon" src={unlockIcon} alt='' />Click here to unlock this locker</a>
+                        </div>
+
+                    </div>
+                    <div className="locker-panel">
+                        <h2>Locker Logs</h2>
+                        <div className="limiter">
+                            <div className="container-table100">
+                                <div className="wrap-table100">
+                                    <div className="table100 ver1 m-b-110">
+                                        <div className="table100-head">
+                                            <table>
+                                                <thead>
+                                                    <tr className="row100 head">
+                                                        <th className="cell100 column1">Status</th>
```

Comment:
Load this from dummy data using string [] for example. Because you will need to do this in the future anyways. This also makes i possible to dynnamically load more table rows instead of having a fixed amount.